.travis.yml

```diff
@@ -82,8 +82,7 @@ before_script:
   - java -version
   - |
     if [ $TOXENV == "es" ]; then
-        sleep 10;
-        curl -v http://localhost:9200/;
+        curl --retry 3 --retry-delay 10 --retry-connrefused http://localhost:9200/;
```

Comment:
`curl: option --retry-connrefused: is unknown` :disappointed: 

It does exist in our image, but travis does not quite match that debian version...

Reply:
https://github.com/mozilla/addons-server/pull/15115/commits/c4fc80877af18a57c1236f05ce21e7ed2094f43d should fix that